pages/linux/bluetoothctl.md

```diff
@@ -0,0 +1,27 @@
+# bluetoothctl
+
+> Handling bluetooth devices from the shell.
+
+- Enter the bluetoothctl shell:
+
+`bluetoothctl`
+
+- List devices:
+
+`bluetoothctl -- devices`
+
+- Pair a device:
+
+`bluetoothctl -- pair {{MAC address}}`
```

Comment:
Let's change "MAC address" to "mac_address" here and everywhere else.

Reply:
Done